captum/attr/_core/modified_relu_grad.py

```diff
@@ -8,8 +8,8 @@
 from .._utils.gradient import apply_gradient_requirements, undo_gradient_requirements
 
 
-class GuidedBackprop(GradientAttribution):
-    def __init__(self, model):
+class ModifiedReluGradientAttribution(GradientAttribution):
+    def __init__(self, model, relu_grad_output=False):
```

Comment:
nit: since `relu_grad_output ` is a bool maybe we can call it `use_relu_grad_output` or something that sounds more bool type ?